src/zjs_net.c

```diff
@@ -0,0 +1,1111 @@
+// Copyright (c) 2017, Intel Corporation.
```

Comment:
Go read my comment at line 770 first because I wrote it first but referred to it later in earlier spots in the file. :)